Use indico instead of a google form to register as participant (#229)
SORSE/sorse.github.io: _faq/about/03-costs.md

---
## Registration

> Registration for each event will be via dedicated registration link on its webpage.

Reviewer: It does not become clear what *on its webpage* means. Do you refer to the page advertising the contribution on the website or do you mean the event pages in indico?

Author: it's the webpage of the event at sorse.github.io, http://www.philipp-s-sommer.de/sorse.github.io/programme/software_demos/event-001/ for instance. any ideas how to rephrase it to make it more clear @eileen-kuehn?

Reviewer: You might simply say the following.

```suggestion
Registration for each event will be via dedicated registration in Indico.
```

To ensure that the following sentence properly describes the data you could then go on with "All registration data will be deleted [...]".

Author: Sounds good. But I think we should still tell them _where_ to find this registration link. Maybe something like
```suggestion
Registration for each event will be via dedicated registration in Indico.
Simply select the event from the [events calendar](){: .missing} and click the 
_Register_ button on its webpage.
```

Reviewer: Great! I like your solution. 
I only compared to the previous version where we also didn't tell them where to go O:)

What I just realised. The first sentence somehow reads strange (maybe just for me as a German?). We could further improve by adding *available*.

```suggestion
Registration for each event will be available via dedicated registration in Indico.
Simply select the event from the [events calendar](){: .missing} and click the 
_Register_ button.
```

Author: Sounds good! What about

```suggestion
Registration for each event will be available via Indico. Select the event from the 
[events calendar](){: .missing} and click the _Register_ button on its webpage.
```

I think that's straight-forward to understand

Reviewer: And another one skipping *on its webpage* :) I think click the button is sufficient.

```suggestion
Registration for each event will be available via Indico. Select the event from the 
[events calendar](){: .missing} and click the _Register_ button.
```

> All data will be deleted from the form after each event. The data will **not** be passed to Zoom or Discord. It will be possible to join the Zoom events via a link and a browser therefore no Zoom account is needed. The link and password to each event will be emailed to the registered participants an hour before the event is due to start.  If you agree for the sponsors to contact you when completing the registration form, the basic information of name and email address will be forwarded to the sponsors.

Reviewer: ```suggestion
All data will be deleted from Indico after each event. The data will **not** be passed to Zoom or Discord. It will be possible to join the Zoom events via a link and a browser therefore no Zoom account is needed. The link and password to each event will be emailed to the registered participants an hour before the event is due to start.  If you agree for the sponsors to contact you when completing the registration form, the basic information of name and email address will be forwarded to the sponsors.
```

Please note, that we cannot promise deletion of all information as we cannot guarantee what happens to the data that was forwarded to sponsors.

## Do I need to pay to attend SORSE events?

No! It’s completely free to the community with the national RSE Groups from many countries collaborating to bring this to you.

> [indico]: https://indico.scc.kit.edu

Reviewer: I propose additionally linking to the category where we will create all the events in, so that visitors can see what is going to happen when.
You can find it at https://indico.scc.kit.edu/category/95/

Author: ```suggestion
```
actually this should be removed, as it's not used anywhere in the text :sweat_smile: 
I added the link to the privacy policy (see https://github.com/SORSE/sorse.github.io/pull/229/files#r455623527). Or shall we add it to the text here as well?

Reviewer: I think it is sufficient to have the proper link in the privacy policies.
